README.md

```diff
@@ -97,15 +94,34 @@ crypto-config/peerOrganizations/org1.example.com/users/User1@org1.example.com/ms
 
 ### Run
 
+#### Binary
+
 Execute `./tape config.yaml 40000` to generate 40000 transactions to Fabric.
 
+#### Docker
+
+```
+docker run -v $PWD:/tmp guoger/tape tape $CONFIG_FILE 40000
+```
+
 *Set this to integer times of batchsize, so that last block is not cut due to timeout*. For example, if you have batch size of 500, set this to 500, 1000, 40000, 100000, etc.
 
+## Tips
+
+- Put this generator closer to Fabric, or even on the same machine. This is to prevent network bandwidth from being the bottleneck. You can use tools like `iftop` to monitor network traffic.
```

Comment:
why not Prometheus？ or ... ?
those some basic level for performance monitoring and some performance env design.
tape will take some cpu resources and make peer maybe not using full cpu resources?

Here is the right place for us to put general performance testing knowledge? or Tape specific Tips?

Reply:
it's just a tip.. and `iftop` is way simpler than `prometheus`. Although i think your comment makes sense - we probably shouldn't make any recommendations for performance monitoring. People have their own choice